services/masterdata/src/lib/core/models/query-model.model.ts

```diff
@@ -0,0 +1,83 @@
+import { HttpParams } from '@angular/common/http';
```

Comment:
This should be a generic model in the app-abstract module imho. User management also has its own implementation. @gitout0fhere what do you think?

Reply:
so, both implementations should be replaced by a generic one

Reply:
It's a generic construct used by the entire backend, so it should be promoted to a generic level.

Reply:
you already have this querymodel it in nswag, generated for each api. I started it's implementation from there, but I've adapted to what I need in my MasterdataCrudHttpService class, as I don't use nswag for masterdata and also didn't create facade classes.

Reply:
I think we should make a decision here: @Johanh-od @jstorteboom, should we promote QueryModel to the LENS framework or is it OK for now and can we promote it later?

Reply:
the QueryModel will remain like this in the masterdata service for now and without using the nswag for now. we could implement this in a future update!